be/src/olap/rowset/segment_v2/inverted_index_compound_directory.cpp

```diff
@@ -349,6 +349,11 @@ void DorisCompoundDirectory::FSIndexInput::readInternal(uint8_t* b, const int32_
     CND_PRECONDITION(_handle->_reader != nullptr, "file is not open");
     std::lock_guard<doris::Mutex> wlock(*_handle->_shared_lock);
 
+    int64_t position = getFilePointer();
```

Comment:
1. Phrase query file pointer is incorrect when skiplist is used
2. Occurs when two terms intersect